app/src/main/java/fr/gaulupeau/apps/Poche/service/tasks/GenericFieldsTask.java

```diff
@@ -37,7 +37,7 @@ public void writeToParcel(Parcel dest, int flags) {
         writeString(genericStringField1, dest);
         writeString(genericStringField2, dest);
 
-        writeBoolean(genericBooleanField1, dest);
+        ParcelCompat.writeBoolean(dest, genericBooleanField1);
```

Comment:
(Boolean)null -> boolean = NPE.

`ParcelCompat.writeBoolean()` works with primitive types, we use object types to have three values.

Reply:
Yeah, I wasn't sure about this myself, and I couldn't test at the moment to make sure.